### ISSUE TITLE
Downloading videos issue

### ISSUE THREAD

**Issue by uahsan3** [LisaAnne/LocalizingMoments#1]
Hi! Thanks for providing the download script. I ran it with the proper flags and it downloads 47 videos to my directory and then stops. The output is this:

> Downlaoding video: 2/10464Could not download link: https://www.flickr.com/video_download.gne?id=4138851955
> Downlaoding video: 14/10464Could not download link: https://www.flickr.com/video_download.gne?id=3844533419
> Downlaoding video: 40/10464Could not download link: https://www.flickr.com/video_download.gne?id=6187277904
> Downlaoding video: 49/10464
> 0 videos are missing

-----------------------------------------------
Please let me know how to download the rest of the videos. Thanks!

**Comment by LisaAnne** [comment]
Well that is embarrassing -- I was debugging the code for someone else and to make it take a shorter period of time I changed it to only download 50 videos.  I then accidentally pushed this code!  This morning I had:

"for dl_link, video_name in zip(link_dict.keys()[:50], link_dict.values())[:50]"

I just changed this to:

"for dl_link, video_name in zip(link_dict.keys(), link_dict.values())"

Hopefully it works now!
